docs/adr/0003-outsource-user-management.md

```diff
@@ -0,0 +1,91 @@
+# 3. Outsource Usermanagement
+
+* Status: accepted <!-- optional -->
+* Deciders: @butonic, @mbarz, @kfreitag, @hd, @pmaier1 <!-- optional -->
```

Comment:
who is hd?

Reply:
holger ... hm need to look up his github nick ... if he has one ;-)

Reply:
if he doesn't have any maybe just add his name in plain text?

Reply:
these are not github nicks
Mine is @micbar 
Klaas: @dragotin 
Holger: @hodyroff

Reply:
fixed